client/src/app/pages/asset-generators/asset-generators.tsx

```diff
@@ -0,0 +1,444 @@
+            style={{
+              backgroundColor: "var(--pf-v5-global--BackgroundColor--100)",
+            }}
+          >
+            <Toolbar {...toolbarProps} clearAllFilters={clearFilters}>
+              <ToolbarContent>
+                <FilterToolbar {...filterToolbarProps} />
+                <ToolbarGroup variant="button-group">
+                  <ToolbarItem>
+                    <CreateButton />
+                  </ToolbarItem>
+                </ToolbarGroup>
+                <ToolbarItem {...paginationToolbarItemProps}>
+                  <SimplePagination
+                    idPrefix="generators-table"
+                    isTop
+                    paginationProps={paginationProps}
+                  />
+                </ToolbarItem>
+              </ToolbarContent>
+            </Toolbar>
+
+            <Table
+              {...tableProps}
+              id="generators-table"
+              aria-label="generators table"
+            >
+              <Thead>
+                <Tr>
+                  <TableHeaderContentWithControls {...tableControls}>
+                    <Th {...getThProps({ columnKey: "name" })} />
+                    <Th {...getThProps({ columnKey: "repository" })} />
+                    <Th {...getThProps({ columnKey: "parameters" })} />
+                    <Th {...getThProps({ columnKey: "values" })} />
+                  </TableHeaderContentWithControls>
+                </Tr>
+              </Thead>
+              <ConditionalTableBody
+                isLoading={isFetching}
+                isError={!!fetchError}
+                isNoData={currentPageItems.length === 0}
+                noDataEmptyState={NO_DATA_EMPTY_STATE}
+                numRenderedColumns={numRenderedColumns}
+              >
+                <Tbody>
+                  {currentPageItems?.map((generator, rowIndex) => (
+                    <Tr key={generator.id} {...getTrProps({ item: generator })}>
+                      <TableRowContentWithControls
+                        {...tableControls}
+                        item={generator}
+                        rowIndex={rowIndex}
+                      >
+                        <Td {...getTdProps({ columnKey: "name" })}>
+                          <Popover
+                            headerContent={generator.name}
+                            bodyContent={`discover applications for ${generator.name} status: ${generator.discoverApplicationsState}`}
+                            headerIcon={
+                              <TaskStateIcon
+                                state={generator.discoverApplicationsState}
+                              />
+                            }
+                            position="top"
+                            id={`generator-name-popover-${generator.id}`}
+                            triggerAction="hover"
+                          >
+                            <Text>{generator.name}</Text>
+                          </Popover>
```

Comment:
I don't think the popover with the task status is needed.  Just the generator's name.